### ISSUE TITLE
SSO at selcont.test.up2university.eu loops

### ISSUE THREAD

**Issue by Dimitrispant** [up2university/single-sign-on#30]
I configured shibboleth at SeLCont test instance like the production instance but it still loops

vhost file https://pastebin.com/L4UviEL3

**Comment by foobarable** [comment]
But in production it worked?

**Comment by Dimitrispant** [comment]
Yes it works on production instance, but it still loops at selcont.test.up2university.eu

**Comment by michzimny** [comment]
@Dimitrispant, do we still have any issues here?

**Comment by Dimitrispant** [comment]
@michzimny , There is still this issue on the test instance, despite I used the same configuration as the production instance.  I am checking Apache logs to see if I can have a clue there, but still nothing. I'm working on it these days.

**Comment by michzimny** [comment]
@Dimitrispant, could you please clarify the current status of this issue?

Am I right in thinking that the goal of this activity is to be able to enable SeLCont to be used as a standalone tool (i.e. without Moodle)? Or, are there also any other purposes behind this?

**Comment by Dimitrispant** [comment]
@michzimny Exactly Michal. The goal of this activity is to enable SeLCont to be used as a standalone tool. If it is used with Moodle, LTI is just fine.

**Comment by michzimny** [comment]
Ok, so, regarding improvements of the Toolbox, I think we should try to solve it out. 

Is it an issue on the SSO side, or can it be on SeLCont's side?

**Comment by Dimitrispant** [comment]
Regarding SelCont side, I have made the same configuration with the production instance (it works on the production instance). I will make some tests later today, just to make sure that it's not from SeLCont's test instance side (I had made tests when I opened this issue, but just to be absolutely sure about it) and I will write the results.

**Comment by foobarable** [comment]
@Dimitrispant Hows the status here?

**Comment by Dimitrispant** [comment]
@foobarable I have sent you an email with some logs from the test instance.
